src/encoded/static/components/static-pages/placeholders/JointAnalysisMatrix.js

```diff
@@ -136,7 +136,13 @@ export class JointAnalysisMatrix extends React.PureComponent {
 
     static defaultProps = {
         'self_results_url'          : '/browse/?experiments_in_set.biosample.biosource_summary=H1-hESC+%28Tier+1%29&experiments_in_set.biosample.biosource_summary=HFFc6+%28Tier+1%29&experiments_in_set.biosample.biosource_summary=H1-hESC+%28Tier+1%29+differentiated+to+definitive+endoderm&experimentset_type=replicate&type=ExperimentSetReplicate&award.project=4DN&limit=all',
-        'encode_results_url'        : 'https://www.encodeproject.org/search/?type=Experiment&biosample_term_name=H1-hESC&biosample_term_name=HFFc6&status!=archived&status!=revoked&limit=all&field=assay_slims&field=biosample_term_name&field=assay_term_name&field=description&field=lab&field=status',
+        'self_results_url_fields'   : [
```

Comment:
These are appended to URL as multiple values for the `field` URI parameter. Back-end then strips all fields from response that aren't present. This is a way to reduce size of AJAXed response without needing to change or re-architect much stuff.